src/helpers/documentation.js

```diff
@@ -72,16 +72,16 @@ export const findInitialOpenTopics = (data, pagePath, rootDir) => {
     initialTopicsState[relPath] = relPath === pgPath;
 
     const pathSegments = relPath.split("/");
+    const currentPathSegments = pgPath.split("/");
 
-    // if path is nested, open the parent dir as well
-    if (pathSegments.length > 2) {
-      const pagePathSegments = pgPath.split("/");
+    if (currentPathSegments.length > 2) {
```

Comment:
It was previously checking to see if all of the parent directories (`data`) instead of current active directory's path. For example, `issuing assets` got 7 items and I click non-nested navigation but because it got a `connecting to outside infrastructure` navigation which has its own nested items, it previously went through this if statement and prevented its navigation to stay open.